lib/optimizely/optimizely_config.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+#    Copyright 2019, Optimizely and contributors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+module Optimizely
+  class OptimizelyConfig
+    def initialize(project_config)
+      @project_config = project_config
+    end
+
+    def optimizely_config
+      experiments_map_object = experiments_map
+      features_map = get_features_map(experiments_map_object)
+      {
+        'experimentsMap' => experiments_map_object,
+        'featuresMap' => features_map,
+        'revision' => @project_config.revision
+      }
+    end
+
+    private
+
+    def experiments_map
+      feature_variables_map = @project_config.feature_flags.reduce({}) do |result_map, feature|
+        result_map.update(feature['id'] => feature['variables'])
+      end
+      @project_config.experiments.reduce({}) do |experiments_map, experiment|
```

Comment:
experiments should include experiments dedicated to "groups" as well. It looks like this list does not include them. Check it out.

Reply:
It does. Project Config already merged grouped experiments in `@experments` array. It contains all experiments except rollouts